src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -87,6 +94,12 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     } else {
       props.updateTrackConfigLabel(selectedCog, !trackOurConfigLabel);
     }
+
+    analyticsTracking.trackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'feature_label_' + (trackOurConfigLabel ? 'on' : 'off')
+    });
```

Comment:
Same as above: map -> forEach, question about `on`/`off`

Reply:
Updated it. Thanks